jerry-core/ecma/operations/ecma-objects.c

```diff
@@ -315,7 +319,7 @@ ecma_op_object_delete (ecma_object_t *obj_p, /**< the object */
     {
       JERRY_ASSERT (false);
```

Comment:
If this is a valid code path, than this should probably be removed. Otherwise, it would be better to use `JERRY_UNREACHABLE`.